cpplint.py

```diff
@@ -61,7 +61,7 @@
 # if empty, use defaults
 _valid_extensions: set[str] = set()
 
-__VERSION__ = "2.0.1"
+__VERSION__ = "2.1.0"
```

Comment:
```suggestion
__VERSION__ = "2.0.1"
```

Reply:
We do want a version bump for the next release, no?